packages/rum/src/domain/assemblyV2.ts

```diff
@@ -63,8 +63,8 @@ export function startRumAssemblyV2(
           ? combine(rumContext, viewContext, actionContext, rawRumEvent)
           : combine(rumContext, viewContext, rawRumEvent)
         const serverRumEvent = combine(
-          savedGlobalContext || getGlobalContext(),
-          customerContext,
+          { context: savedGlobalContext || getGlobalContext() },
+          { context: customerContext },
           withSnakeCaseKeys(rumEvent)
```

Comment:
This could be simpler:
```ts
const serverRumEvent = withSnakeCaseKeys(rumEvent)
serverRumEvent.context = combine(savedGlobalContext || getGlobalContext(), customerContext)
```